### ISSUE TITLE
Lowercasing namespacing and chip namespace

### ISSUE THREAD

**Issue by CvRXX** [kvasir-io/HardwareDescriptionFileGenerator#5]
@odinthenerd and I really want all namespaces to be lowercase from now on. Could you implement that?

Also since all the files generated here are chip specific I think we should move it all to the kvasir::chip namespace. Do you agree?

**Comment by odinthenerd** [comment]
yeah it was probably a stupid idea to UpperCammelCase namespaces back in the day, doesn't seem natural to anyone but me ;)

**Comment by CvRXX** [comment]
It kind of ruins IDE

**Comment by jackieokay** [comment]
Yeah, the camelcase namespaces were bothering me.

**Comment by jackieokay** [comment]
Currently, structs defined by IO extensions are in the "Kvasir::Io" namespace ([example](https://github.com/kvasir-io/Kvasir/blob/master/Lib/Chip/CM4/Freescale/MK11D5WS/Io.hpp)). In the new scheme, should everything be in the "kvasir::chip" namespace?

**Comment by odinthenerd** [comment]
Yes they should move to kvasir::chip::io
